index.js

```diff
@@ -10,61 +12,74 @@ if (localStorage.getItem('books') !== null) {
 
   getbook.forEach((item) => {
     addBook.innerHTML += `
-      <div>
-        <p>${item.title}</p>
-        <p>${item.author}</p>
+      <div class="books">
+        <p>"${item.title}" by "${item.author}"</p>
         <button class="remove" name="${item.title}">Remove</button>
-        <hr>
       </div>
     `;
   });
 }
 /* ***** */
 
-function Book(title, author) {
-  this.title = title;
-  this.author = author;
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
 }
 
-addButton.addEventListener('click', (e) => {
-  e.preventDefault();
-
-  addBook.innerHTML += `
-    <div>
-      <p>${title.value}</p>
-      <p>${author.value}</p>
+class Methods {
+  static addbook(book1) {
+    addBook.innerHTML += `
```

Comment:
- [ ] It would be better to wrap all your objects and arrays inside one class. and make sure your add and remove methods are all working fine.